### PR TITLE
ACX Everywhere Spotlight

### PR DESCRIPTION
**Pull request by Raemon**
This swaps out Replacing Guilt with ACX Everywhere.

It also introduces the Post type to the CuratedContentItem, and moves the hardcoded curated contents over into a separate file.

It ALSO fixes a bug in LinkCard that was preventing link cards from being clickable in some cases.

image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1202872066800252) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
This looks to me like a draft PR

**Comment by Raemon** [comment]
Sorry, was about to edit in a description when we had another minor emergency.

File: packages/lesswrong/components/recommendations/curatedContents.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import { CuratedContent } from "./CuratedContentItem"
```
Comment (jpaddison3) [inline packages/lesswrong/components/recommendations/curatedContents.ts:1]:
```suggestion
import type { CuratedContent } from "./CuratedContentItem"
```

**Review by jpaddison3 — COMMENTED** [review]
No objection from the EA Forum

**Comment by Raemon** [comment]
image

**Review by darkruby501 — APPROVED** [review]

**Comment by Raemon** [comment]
image

**Pull request merged**